openadapt/record.py

```diff
@@ -646,7 +751,22 @@ def on_press(event_q, key, injected):
                 logger.info("Stop sequence entered! Stopping recording now.")
                 stop_sequence_detected = True
 
-    def on_release(event_q, key, injected):
+    def on_release(
+        event_q: queue.Queue,
+        key: keyboard.KeyboardEvent,
```

Comment:
`key: Union[keyboard.Key, keyboard.KeyCode]` according to the [documentation](https://pynput.readthedocs.io/en/latest/keyboard.html)

(also add `from typing import Union` at the top)

Reply:
resolved in https://github.com/OpenAdaptAI/OpenAdapt/pull/289/commits/87188ec08861da59a980d007fa64efda02f222e8